dev/django.Dockerfile

```
@@ -0,0 +1,19 @@
+FROM python:3.8-slim
+# Install system librarires for Python packages:
+# * psycopg2
+RUN apt-get update && \
+    apt-get install --no-install-recommends --yes \
+        libpq-dev gcc libc6-dev && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# Only copy the setup.py, it will still force all install_requires to be installed,
+# but find_packages() will find nothing (which is fine). When Docker Compose mounts the real source
+# over top of this directory, the .egg-link in site-packages resolves to the mounted directory
+# and all package modules are importable.
+COPY ./setup.py /opt/django/setup.py
+RUN pip install --editable /opt/django/
```

```suggestion
COPY ./requirements.txt /opt/dandi-publish/requirements.txt
RUN pip install --requirement /opt/dandi-publish/requirements.txt
```
The pip dependancies are listed in `requirements.txt`, `setup.py` doesn't exist.

---

We should switch to a `setup.py`. This may need to be rebased, I think @dchiquit did so in #12 .